lib/exe/fetch.php

```diff
@@ -96,7 +105,15 @@
     // finally send the file to the client
     $evt = new Event('MEDIA_SENDFILE', $data);
     if($evt->advise_before()) {
-        sendFile($data['file'], $data['mime'], $data['download'], $data['cache'], $data['ispublic'], $data['orig']);
+        sendFile(
```

Comment:
Though not CSP per se `X-Frame-Options: SAMEORIGIN` could be added as a header, alternatively `frame-ancestors: 'self'` could be added to the CSP 

ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors